src/Css/Processor.php

```diff
@@ -24,6 +24,21 @@ public function getRules($css, $existingRules = array())
         return $rulesProcessor->convertArrayToObjects($rules, $existingRules);
     }
 
+    /**
+     * Get all media queries from the given css string. Returns an empty array
+     * when non were found.
+     *
+     * @param string $css
+     * @return array
```

Comment:
please document it as `string[]` to be more precise.

Also, please separate the param and return annotation for consistency.